analyzer-java/src/main/java/com/infosupport/ldoc/analyzerj/AnalysisVisitor.java

```diff
@@ -384,13 +399,13 @@ public List<Description> visit(MethodCallExpr n, Analyzer arg) {
 
     List<Description> arguments = new ArrayList<>();
     for (Expression argument : n.getArguments()) {
-      String type = resolver.calculateType(argument).describe();
+      String type = resolve(argument);
       String text = argument.toString();
       arguments.add(new ArgumentDescription(type, text));
     }
     return List.of(
         new InvocationDescription(
-            n.getScope().map(s -> resolver.calculateType(s).describe()).orElse("?"),
+            n.getScope().map(this::resolve).orElse("?"),
```

Comment:
This is now inconsistent - is the fallback string `"Unknown"` or `"?"`? Prefer making them the same - and ideally consistent with upstream, I think there's a `"?"` in `pitstop.analyzed.json`.